Fixes #9 - Close command

AntiD2ta opened the pull request:
Fix #9 

Changes:
- Change project structure
  - Add module `src` and put within `bot` and `model` modules
- Add `close` command

AntiD2ta committed:
refs #9 - Change project structure. Add `src` module

AntiD2ta committed:
[bot] refs #9 - Add close command

e1Ru1o commented on src/bot/handlers/close.py:
```diff
@@ -0,0 +1,31 @@
+    try:
+        assert is_chat_admin(context.bot, chat, user), ADMINS_ONLY
+        assert context.chat_data.get('active'), ACTIVE
+        assert context.chat_data.get('voters'), NO_VOTES
```
```suggestion
        assert any(context.chat_data.get('voters', dict()).values()), NO_VOTES
```
Check if at least one voter has valid votation

e1Ru1o commented on src/bot/handlers/close.py:
```diff
@@ -0,0 +1,31 @@
+        assert context.chat_data.get('active'), ACTIVE
+        assert context.chat_data.get('voters'), NO_VOTES
+
+        votes = [vote for vote in context.chat_data['voters'].values()]
```
```suggestion
        votes = context.chat_data['voters'].values()
```

AntiD2ta committed:
[bot] refs #9 - Update src/bot/handlers/close.py

Co-authored-by: Lazaro Raul <47638426+stdevRulo@users.noreply.github.com>

AntiD2ta committed:
[bot] refs #9 - Update src/bot/handlers/close.py

Check if a non `None` vote exist in `voters`

Co-authored-by: Lazaro Raul <47638426+stdevRulo@users.noreply.github.com>

e1Ru1o committed:
[model] - Use inf as the distance of the initial answer of the model

e1Ru1o committed:
[src] - Some changes

Changes:
- Added solve function of model module to src module
- import Scheduler_Bot directly from src instead of src.bot
